src/ops/control_flow.jl

```diff
@@ -250,6 +250,7 @@ Example using shape_invariants:
 """
 @op function while_loop(condition, body, variables; name=nothing, shape_invariants=nothing,
                         parallel_iterations=10, back_prop=true, swap_memory=false)
+    error("Dynamic while loops are not currently ipmlemented.")
     g = Graph()
```

Comment:
Maybe comment out the body with a multiline comment (after the error),
so that synax highlighter will make it clear that that code is not being run?